docs/auth/index.md

```diff
@@ -77,7 +77,6 @@ built-in providers:
 +
 const app = createApp({
   apis,
-  plugins: Object.values(plugins),
 +  components: {
```

Comment:
Not part of the PR, but this actually has the wrong indentation (or rather, the other parts of the diff block should be indented more). Same below